sql/moz-fx-data-shared-prod/telemetry/agg_active_users/view.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.agg_active_users`
+AS
+SELECT
+  *
+FROM
+  `moz-fx-data-shared-prod.telemetry_derived.agg_active_users_v1`
```

Comment:
We already have a [few aggregate tables](https://github.com/mozilla/bigquery-etl/tree/main/sql/moz-fx-data-shared-prod/telemetry_derived/experiment_enrollment_aggregates_v1) that all follow this naming schema:

```suggestion
  `moz-fx-data-shared-prod.telemetry_derived.active_users_aggregates_v1`
```